packages/website/blog/2024-05-11-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -0,0 +1,310 @@
+
+> ⏳ indicates a change that has been scheduled for v8 but not yet released.
+> We'll update this blog post as the corresponding pull requests land.
+
+### ⏳ ESLint v9 Support
+
+typescript-eslint v8 ships with full support for ESLint v9.
+
+typescript-eslint v7 was our first version that supported ESLint's [new "flat" config file format](https://eslint.org/docs/latest/use/configure/configuration-files), which was already available in ESLint v8.
+ESLint v9 still supports ESLint's [older legacy config file format](https://eslint.org/docs/latest/use/configure/configuration-files-deprecated) so our tooling does as well.
+However, ESLint v9 also includes a set of breaking changes that we added support for in typescript-eslint v8.
+See the [ESLint v9 release blog post](https://eslint.org/blog/2024/04/eslint-v9.0.0-released) and for more details.
+
+### ⏳ Project Service
+
+The biggest new feature added in this version is the stability of our new "project service".
+In short, the project service is a new way to enable [typed linting](/getting-started/typed-linting) that is generally _easier to configure_ and _faster at runtime_ than our previous offerings.
+It's been experimentally available since v6.1.0 under the name `EXPERIMENTAL_useProjectService`; now, we've renamed it to `projectService`.
+
+You can use the new project service in your configuration instead of the previous `parserOptions.project`:
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: true,
+        // Add line
+        projectService: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+);
+```
+
+The project service will automatically find the closest `tsconfig.json` for each file (like `project: true`).
+It also allows enabling typed linting for files not explicitly included in a `tsconfig.json`.
+This should remove the need for custom `tsconfig.eslint.json` files to lint files like `eslint.config.js`!
+
+Typed linting for out-of-project files can be done by specifying two properties of a `parserOptions.projectService` object:
+
+- `allowDefaultProjectForFiles`: a glob of a small number of out-of-project files to enable a slower default project on
+- `defaultProject`: path to a TypeScript configuration file to use for the slower default project
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: ['packages/*/tsconfig.json', 'tsconfig.eslint.json'],
```

Comment:
This would be a glob right?
```suggestion
        project: ['packages/**/tsconfig.json', 'tsconfig.eslint.json'],
```

Reply:
It wouldn't: https://typescript-eslint.io/troubleshooting/performance-troubleshooting#wide-includes-in-your-eslint-options

Part of what I'm excited for with the project service is to no longer need to remember to use globs or not!